renderdoc/api/replay/renderdoc_replay.h

```diff
@@ -1579,6 +1579,10 @@ extern "C" RENDERDOC_API void RENDERDOC_CC RENDERDOC_CheckAndroidPackage(const c
                                                                          const char *exe,
                                                                          AndroidFlags *flags);
 
+DOCUMENT("Internal function that attempts to push Vulkan layer to Android application.");
+extern "C" RENDERDOC_API bool RENDERDOC_CC RENDERDOC_PushLayerToAndroidApp(const char *host,
+                                                                           const char *exe);
```

Comment:
In my head, the APK is the bundle that can be passed around as a file and contains enough bits to get your installation started, while the app is the actual installed program that primarily contains components extracted from the APK (i.e. shared objects per target ABI) and items downloaded from various servers after installation (assets, etc.).  So it is sort of an extended subset of the APK... but it happens to also contain the original APK (base.apk), which is nice.

I agree the name should clearly identify what it is doing.  I could expand the function name to RENDERDOC_PushLayerToInstalledAndroidApp?